src/cicero/dom.cto

```diff
@@ -146,9 +147,29 @@ asset List extends Section {
 }
 
 /**
- * An inline function can be included in a contract to perform basic calculations
+ * Ergo inlined in contracts or clauses
+ */
+abstract asset Ergo extends Element {
```

Comment:
Need a better name for this, as Ergo is the name of the language. E.g. `Script` ?

Reply:
I like `Script`.